google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoManagedVmDeployment.java

```diff
@@ -138,6 +140,8 @@ public void run() {
       @Override
       public void processTerminated(ProcessEvent event) {
         if (event.getExitCode() == 0) {
+          UsageTrackerProvider.getInstance().trackEvent(GctTracking.CATEGORY, GctTracking.DEPLOY, "deploy.flex.java", null);
```

Comment:
It could be interesting to track whether it was a jar/war deployment and auto vs custom.